debian/control

```diff
@@ -9,6 +9,6 @@ Vcs-Browser: https://github.com/open-switch/opx-tools
 Vcs-Git: https://github.com/open-switch/opx-tools.git
```

Comment:
Should you also be defining additional Build-Depends?

Reply:
These are python scripts, it may not be needed.